src/graphnet/data/i3extractor.py

```diff
@@ -296,6 +296,7 @@ def __call__(self, frame, padding_value=-1) -> dict:
                 'pid': MCInIcePrimary.pdg_encoding,
                 'interaction_type': interaction_type,
                 'elasticity': elasticity,
+                'dbang_decay_length': self.__extract_dbang_decay_length__(frame, padding_value),
```

Comment:
I think the change in this file makes a lot of sense. However, the naming of this method (which, I realise, was not implemented in this PR) is non-standard. Method names that start and end in double underscores are typically reserved for special, built-in methods (e.g., `__init__`, `__call__`, `__repr__`), and I would suggest we adhere to that. If we want to signify that this is meant to be a private/protected method, then it could be called `_extract_dbang_decay_length`, i.e., with a single leading underscore, which is what has been done elsewhere in the repo.